bb_wrapper/wrapper/pagamento_lote.py

```diff
@@ -481,3 +482,119 @@ def consultar_pagamento_tributo(self, _id):
         response = self._get(url)
 
         return response
+
+    ################
+    #     PIX      #
+    ################
+
+    def _criar_dados_transferencia_pix(
+        self,
+        n_requisicao,
+        agencia,
+        conta,
+        dv_conta,
+        data_transferencia,
+        valor_transferencia,
+        forma_id,
+        ddd,
+        telefone,
+        email,
+        cpf,
+        cnpj,
+        chave_aleatoria,
```

Comment:
Seria melhor fazer algo mais inteligente em cima dessas chaves e `forma_id`.


Recomendo criar um model enum tipo

```python

TipoChavePIX(Enum):
    telefone = 'telefone'
    documento = 'cpf/cnpj'
    email = 'email'
    chave_aleatoria = 'chave_aleatoria'

```

Com isso, fica mais explicito e simples os argumentos da função.

E diria que da para manter o mesmo tipo para cpf/cnpj. Pois tem como identificar de forma simples se é um CPF ou se é um CNPJ!


```suggestion
        tipo_chave,
        chave,
```